Sources/Swift/Integrations/SessionReplay/SentrySessionReplay.swift

```diff
@@ -130,6 +130,16 @@ class SentrySessionReplay: NSObject {
         videoSegmentStart = nil
         displayLink.link(withTarget: self, selector: #selector(newFrame(_:)))
     }
+    
+    func stop() {
+        lock.lock()
+        defer { lock.unlock() }
+        displayLink.invalidate()
+        if isFullSession {
+            prepareSegmentUntil(date: dateProvider.date())
+        }
```

Comment:
This is essentially `self.pause()` except for the last line in pause that sets `isSessionPaused = false`, could we reuse it? Although I would expect `pause()` to set `isSessionPaused = true`, I don't understand what's going on there.

Reply:
I forget the stop function in here. I will remove it.